lib/utilities/platform-checker.js

```diff
@@ -2,33 +2,50 @@
 
 var semver = require('semver');
 var logger = require('heimdalljs-logger')('ember-cli:platform-checker:');
+var loadConfig = require('./load-config');
 
-var LOWER_RANGE = '0.12.0';
-var UPPER_RANGE = '7.0.0';
+var testedEngines;
+if (process.platform === 'win32') {
+  testedEngines = loadConfig('appveyor.yml')
+    .environment.matrix
+    .map(function(element) {
+      return element.nodejs_version;
+    })
+    .join(' || ');
+} else {
+  testedEngines = loadConfig('.travis.yml')
```

Comment:
Do we actually include these files in our published tarballs?  I'm not opposed, but I would be surprised if they were not ✂️ in the recent march to reduce file counts...